docs/tutorials/mongoid-documents.txt

```diff
@@ -177,22 +177,19 @@ Types that are not supported as dynamic attributes since they cannot be cast are
 - ``DateTime``
 - ``Range``
 
-
 .. _stringified-symbol:
 
 The StringifiedSymbol Type
 --------------------------
 
 The ``StringifiedSymbol`` field type is the recommended field type for storing
-values that should be exposed as symbols to Ruby applications.
-
-When using the ``Symbol`` field type, Mongoid defaults to storing the values
-as BSON symbols. However, the BSON symbol type is deprecated and working with
-BSON symbols is difficult in programming languages that do not have a native
-symbol type. The ``StringifiedSymbol`` field type stores values as strings
-in MongoDB, thus enabling easy interoperability with software written in other
-programming languages, but exposes the values as instances of ``Symbol`` in
-Ruby programs.
+values that should be exposed as symbols to Ruby applications. When using the ``Symbol`` field type,
+Mongoid defaults to storing values as BSON symbols. For more information on the
+``BSON::Symbol`` type, see :ref:`here <bson-symbol>`.
+However, the ``BSON::Symbol`` type is deprecated and is difficult to work with in programming languages
+without native symbol types, so the ``StringifiedSymbol`` type allows the use of symbols
```

Comment:
Same here, other languages do not have a `BSON::Symbol` type since this type is Ruby driver-specific.